lib/tasks/themes.rake

```diff
@@ -194,3 +194,43 @@ ensure
   db&.remove
   redis&.remove
 end
+
+desc "Clones all official themes."
+task "themes:clone_all_official" do |task, args|
+  FileUtils.rm_rf("tmp/themes")
+
+  official_themes =
+    Set
+      .new(
+        %w[
+          discourse-brand-header
+          discourse-category-banners
+          discourse-clickable-topic
+          discourse-color-scheme-toggle
+          discourse-custom-header-links
+          Discourse-easy-footer
+          discourse-gifs
+          discourse-topic-thumbnails
+          discourse-search-banner
+          discourse-unanswered-filter
+          discourse-versatile-banner
+          DiscoTOC
+          unformatted-code-detector
```

Comment:
Just a standard list for now and we can continue to add to this over time.

Reply:
Wondering if we want to use something like our `metadata.rb` for plugins

Reply:
Done :) I moved it to `ThemeMetadata::OFFICIAL_THEMES`